### ISSUE TITLE
[Bug]: Mohist/Magma - ProRPGItems not loading, causes error message

### ISSUE THREAD

**Issue by JakeBat-H** [promcteam/proskillapi#389]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Have you tried to reproduce this without OP?

- [X] I have tested without OP

### Current Behavior

See title. See error below.
[message.txt](https://github.com/promcteam/proskillapi/files/9639626/message.txt)

### Expected Behavior

Plugin loads normally, does not give error. 

### Environment

```markdown
- Server Software:
- ProSkillAPI Version:
- ProMCCore Version:
Plugins (15): LuckPerms, PlaceholderAPI, Vault, WorldEdit, DeluxeTags, RiseCore, WorldGuard, BanItem, CMILib, ProMCCore, CMI, ProSkillAPI, ProSkillAPIParties, SCore, ExecutableItems
```

### latest.log

[latest (1).log](https://github.com/promcteam/proskillapi/files/9639627/latest.1.log)

**Comment by Travja** [comment]
What MC version does this target? And what Magma/Mohist version?

**Comment by Travja** [comment]
I downloaded the latest version of Mohist and attempted to use that to run the plugin. It appears that even though the `libraries` plugin.yml functionality in Spigot appears to have been implemented in 1.16.5, it is not available in Mohist. I've altered ProMCCore to shade the mysql-connector that ProRPGItems was wanting and that appears to resolve any issues on my end. You'll have to confirm if your error still exists or not, but for now I'm going to close this out.
